client/src/components/EventView/Description/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styled from "styled-components";
+
+const TextBar = styled.div`
+font-size: 2.2vw;
+font-weight: bold;
+padding: 2% 0%;
+`;
+
+//takes in a description
+const Description = ({description,...props}) => {
+return <div>
+    <TextBar>Description</TextBar>
+    <div>{description}</div>
```

Comment:
The description text is a little small, maybe set the font size to be fontSize20.